doc/mgmt/OpenConfig_Platform.md

```diff
@@ -0,0 +1,454 @@
+		<td>TRANSCEIVER_INFO</td>
+		<td>manufacturer</td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left:5em;">vendor-part</th>
+		<td>State_DB</td>
+		<td>TRANSCEIVER_INFO</td>
+		<td>vendor_oui</td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left:5em;">serial-no</th>
+		<td>State_DB</td>
+		<td>TRANSCEIVER_INFO</td>
+		<td>serial</td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left:5em;">date-code</th>
+		<td>State_DB</td>
+		<td>TRANSCEIVER_INFO</td>
+		<td>vendor_date</td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left:5em;">supply-voltage/instant</th>
+		<td>State_DB</td>
+		<td>TRANSCEIVER_DOM_SENSOR</td>
+		<td>voltage</td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left: 4em;">physical-channels</th>
+		<td></td>
+		<td></td>
+		<td></td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left: 5em;">channel</th>
+		<td></td>
+		<td></td>
+		<td></td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left: 6em;">state</th>
+		<td></td>
+		<td></td>
+		<td></td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left:7em;">laser-temperature/instant</th>
+		<td>State_DB</td>
+		<td>TRANSCEIVER_DOM_SENSOR</td>
+		<td>temperature</td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left:7em;">output-power/instant</th>
+		<td>State_DB</td>
+		<td>TRANSCEIVER_DOM_SENSOR</td>
+		<td>tx1power, tx2power, tx3power, tx4power</td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left:7em;">input-power/instant</th>
+		<td>State_DB</td>
+		<td>TRANSCEIVER_DOM_SENSOR</td>
+		<td>rx1power, rx2power, rx3power, rx4power</td>
+	</tr>
+	<tr>
+		<th align="left" style="padding-left:7em;">laser-bias-current/instant</th>
+		<td>State_DB</td>
+		<td>TRANSCEIVER_DOM_SENSOR</td>
+		<td>tx1bias, tx2bias, tx3bias, tx4bias</td>
+	</tr>
+</table>
+
+# 5 Error Handling
+Invalid operations will report an error.
+# 6 Unit Test cases
+## 6.1 Functional Test Cases
+Operations: 
+gNMI- Get / Subscribe
+REST - GET
+
+1.	Verify that operations supported for gNMI/REST works fine for serial-no of transceiver.
+2.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:connector-type.
+3.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:vendor.
+4.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:vendor-part.
+5.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:serial-no.
+6.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:date-code.
+7.	Verify that operations supported for gNMI/REST works fine for supply-voltage of transceiver.
+8.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:laser-temperature.
+9.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:output-power.
+10.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:input-power.
+11.	Verify that operations supported for gNMI/REST works fine for oc-transceiver:laser-bias-current.
+
```

Comment:
Add test cases for breakout as well. If a transceiver is in a breakout mode, like 4x100G, need to make sure the display is handled correctly.